web/migrations/versions/ca00ec32581b_.py

```diff
@@ -25,7 +24,7 @@
 
 
 def upgrade():
-    db.engine.execute(
+    op.execute(
```

Comment:
instead of using plain SQL may be you can try similar to `web/migrations/versions/81c7ffeffeee_.py`.